### PR TITLE
Update PartiallySupportedFlowKeywords.cpp and STOPs

### PR DESCRIPTION
**Pull request by OPMUSER**
Updates to partially supported keywords and enforced STOPs.

### CODE REVIEW

File: opm/simulators/utils/PartiallySupportedFlowKeywords.cpp [hunk 1]
```diff
@@ -85,13 +85,13 @@ partiallySupported()
          {
             "GCONINJE",
             {
-               {10,{false, allow_values<std::string> {"RATE", "NETV", "RESV", "VOID"}, "GCONINJE(GUIPHASE): only RATE/NETV/VOID are supported - will STOP"}}, // GUIDE_RATE_DEF
+               {10,{false, allow_values<std::string> {"RATE", "NETV", "RESV", "VOID"}, "GCONINJE(GUIPHASE): only RATE/NETV/RESV/VOID are supported - will STOP"}}, // GUIDE_RATE_DEF
```
Comment (alfbr) [inline opm/simulators/utils/PartiallySupportedFlowKeywords.cpp:88]: Either you need to remove the "will STOP" message, or make sure the simulator stops with "true".

File: opm/simulators/utils/PartiallySupportedFlowKeywords.cpp [hunk 2]
```diff
@@ -85,13 +85,13 @@ partiallySupported()
...
             },
          },
          {
             "GCONPROD",
             {
-               {2,{false, allow_values<std::string> {"NONE", "FLD", "ORAT", "WRAT", "GRAT", "LRAT", "RESV"}, "GCONPROD(TARGET): valid option should be FLD/ORAT/WRAT/GRAT/LRAT or RESV – will STOP"}}, // CONTROL_MODE
+               {2,{false, allow_values<std::string> {"NONE", "FLD", "ORAT", "WRAT", "GRAT", "LRAT", "RESV"}, "GCONPROD(TARGET): valid option should be NONE/FLD/ORAT/WRAT/GRAT/LRAT or RESV – will STOP"}}, // CONTROL_MODE
```
Comment (alfbr) [inline opm/simulators/utils/PartiallySupportedFlowKeywords.cpp:94]: Either you need to remove the "will STOP" message, or make sure the simulator stops with "true".

File: opm/simulators/utils/PartiallySupportedFlowKeywords.cpp [hunk 3]
```diff
@@ -107,7 +107,7 @@ partiallySupported()
          {
             "GRIDOPTS",
             {
-               {1,{false, allow_values<std::string> {"NO"}, "GRIDOPTS(TRANMULT): only the NO option is supported – value ignored"}}, // TRANMULT
+               {1,{false, allow_values<std::string> {"NO", "YES"}, "GRIDOPTS(TRANMULT): should be set to either NO or YES – will STOP"}}, // TRANMULT
```
Comment (alfbr) [inline opm/simulators/utils/PartiallySupportedFlowKeywords.cpp:110]: Either you need to remove the "will STOP" message, or make sure the simulator stops with "true".

**Comment by OPMUSER** [comment]
Closing as superseded #3738

**Pull request closed without merging**